core/src/main/scala/cats/instances/try.scala

```diff
@@ -75,9 +75,10 @@ trait TryInstances extends TryInstances1 {
         ta match { case Success(a) => Try(map(a)); case Failure(e) => Try(recover(e)) }
 
       override def redeemWith[A, B](ta: Try[A])(recover: Throwable => Try[B], bind: A => Try[B]): Try[B] =
-        try ta match {
-          case Success(a) => bind(a); case Failure(e) => recover(e)
-        } catch { case NonFatal(e) => Failure(e) }
+        try
+          ta match {
+            case Success(a) => bind(a); case Failure(e) => recover(e)
+          } catch { case NonFatal(e) => Failure(e) }
```

Comment:
I wonder why if scalafmt says "A" it does not say "B" making it formatted in this way:
```
try
  ta match {
    case Success(a) => bind(a); case Failure(e) => recover(e)
  }
catch { case NonFatal(e) => Failure(e) }
```
Because to me the current reformatting looks more weird and hard-to-read than it was before.

Reply:
Asked about it in the scalafmt channel:
https://discord.com/channels/632642981228314653/632665341864181780/923692567370022962

Reply:
I would say that is the correct format inline with braceless syntax. Interesting that a minor point release would do that.

Reply:
It does not seem correct to me. In fact, when I first saw the change, I got surprised because I didn't find the `catch` block for the `try` right away. Because the way how scalafmt formats it may now create an impression that `catch` belongs to somewhere else deep inside rather than the outermost `try`. Similar to:
```
if (checkCondition())
  ta match { 
    case Success(a) => bind(a); case Failure(e) => recover(e)
  } else doSomethingOtherwise()
```
looks quite confusing but
```
if (checkCondition())
  ta match { 
    case Success(a) => bind(a); case Failure(e) => recover(e)
  }
else doSomethingOtherwise()
```
looks way clearer and easier to read.

Reply:
Sorry, I was looking at the `try` where it wrapped `ta match {`. what happens if you wrap the `catch` to the next line, does it stay?

Reply:
Side note: I really wish scalafmt would stop changing the format slightly on every release.

Reply:
@ekrich, nope – even if I wrap `catch` to the next line manually, it returns back after reformatting:
```
--- a/core/src/main/scala/cats/instances/try.scala
+++ b/core/src/main/scala/cats/instances/try.scala
@@ -78,8 +78,7 @@ trait TryInstances extends TryInstances1 {
         try
           ta match {
             case Success(a) => bind(a); case Failure(e) => recover(e)
-          }
-        catch { case NonFatal(e) => Failure(e) }
+          } catch { case NonFatal(e) => Failure(e) }
```

Reply:
Then I feel that is ugly - I didn't look at your settings but still. It should line up the `try` and `catch`.

Reply:
it's a bug. would be nice to have an issue submitted in github, works better than asking in discord (which i visit rarely if ever).

Reply:
> Side note: I really wish scalafmt would stop changing the format slightly on every release.

most of the changes are configurable. for instance, in v3.2.2, binpacked code can finally use `danglingParentheses` (it was ignored before); but if you don't want that, disable `danglingParentheses` (or enable `optIn.configStyleArguments`).

Reply:
fixed in 3.3.0.

Reply:
@kitbellew personally I don't want to configure it. I just want it to have a good standard style even if it isn't my first choice and I want that configuration to not require updates. Instead my CIs have been going red because updating scalafmt has recently changed either the formatting or the configuration file or both. I appreciate the work people have given on the project, I just wish it were more conservative so that unless you get a major version bump your previous config will be valid and the format won't change (unless you opt in to new versions).

Anyway this is OT for here but indulged myself. Apologies.

Reply:
if nothing changes... why do you need to upgrade?

Reply:
So I wonder – if it supposed to be fixed in version 3.3.0 (which just has been released) then should we just skip this one and close this PR?

Reply:
@kitbellew we upgrade because scala-steward sends the update and there may be bug fixes or security fixes. Those are the motivations I think most libraries have for updating.